specification/configuration.md

```diff
@@ -130,6 +136,31 @@ beyond the OpenTelemetry specification exist.
   - Distribution MUST default to `otlp`
   - Distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
 
+### Real User Monitoring Libraries
+
+Real User Monitoring (RUM) instrumentation libraries cannot use environment variables for configuration.
+Instead, they MUST expose a `SplunkRum` object/class/namespace (depending on the language used) that allows
+setting the following properties (in a language-specific way: Java may use builders, Swift can use named
+parameters with default values, JavaScript can use objects, etc.):
+
+| Configuration property (default value) | Description                                                                                                                                                                                                 | Required |
+| -------------------------------------- | -----------                                                                                                                                                                                                 | -------- |
+| `beaconUrl` ()                         | RUM beacon URL, ex. `https://rum-ingest.<realm>.signalfx.com/v1/rum`. [1] [2]                                                                                                                               | Yes      |
```

Comment:
1) Is there a reason this doesn't default to us0 endpoint? 2) Could we have an alternative `beaconEndpoint = 'us0' | 'us1' | ...`, where if you set `beaconEndpoint` then you don't have to set `beaconUrl`? `beaconUrl` would still be available.

Reply:
It's a good idea, the collector already does something similar with realm/ingest URL. I'll add it, thanks!